ci/docker/generator/Dockerfile

```diff
@@ -1,22 +1,20 @@
-FROM golang:1.20.1-buster
+FROM ghcr.io/nordsecurity/nordvpn-linux/golang:debian-buster-20240201
 
 LABEL org.opencontainers.image.source=https://github.com/NordSecurity/nordvpn-linux
 
-ENV GOBIN=/usr/bin
+RUN apt-get update; \
+    apt-get -y --no-install-recommends install unzip; \
+    apt-get clean; \
+    rm -rf /var/lib/apt/lists/*
 
-RUN apt-get update \
-    && apt-get install -y zip \
-    # cleanup
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN wget --progress=dot:giga https://github.com/protocolbuffers/protobuf/releases/download/v21.6/protoc-21.6-linux-x86_64.zip -O /tmp/protoc.zip && \
```

Comment:
`RUN` directives can be minimized, because each `RUN` creates another Docker layer and increases image size.